client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEdit.tsx

```diff
@@ -264,11 +264,11 @@ export const RequestLineEdit = ({
       />
 
       {line && (
-        <Box padding={'2px 16px 0 16px'}>
+        <>
           {plugins.requestRequisitionLine?.editViewInfo?.map((Info, index) => (
             <Info key={index} line={line} requisition={requisition} />
           ))}
-        </Box>
+        </>
```

Comment:
Cool -- I don't think you even need the fragment tags since it's just a single node. Seems to be fine with just:
```tsx
      {line &&
        plugins.requestRequisitionLine?.editViewInfo?.map((Info, index) => (
          <Info key={index} line={line} requisition={requisition} />
        ))}